packages/api-explorer/src/scenes/OAuthScene/OAuthScene.tsx

```diff
@@ -72,9 +77,7 @@ export const OAuthScene: React.FC<OAuthSceneProps> = ({ sdk }) => {
           </Text>
         </>
       )}
-      {!loading && sdk && sdk.authSession.isAuthenticated() && (
-        <Redirect to={newUrl} />
-      )}
+      {!loading && auth && auth.isAuthenticated() && history.push(newUrl)}
```

Comment:
I think `history.push()` is working but it's not matching any of our defined routes. 

I also think that this line can be removed and replaced with `history.push(newUrl)` on line 66. That way, we reserve the value of the return statement for JSX content while the component logic resides in the useEffect hook.